theme/static/js/hs_file_browser.js

```diff
@@ -596,19 +596,23 @@ function bindFileBrowserItemEvents() {
             }
             menu = $("#right-click-menu");
 
-            var fileAggType = $(event.target).closest("li").find("span.fb-logical-file-type").attr("data-logical-file-type");
+            fileAggType = [];
+            // main-file is available on the aggregation folder and only single file aggregations have a data-pk of 1 on the file
+            if ($(event.target).closest("li").attr("main-file") || $(event.target).closest("li").attr("data-pk") === "1"){
+                var fileAggType = $(event.target).closest("li").find("span.fb-logical-file-type").attr("data-logical-file-type");
+            }
```

Comment:
If ```fileAggType``` is initialized before, see my comment above, it does not have to be redefined here and can just be assigned a value here, and I assume the returned value should be a array type to conform with the variable initialization type.